apps/music/views/albums/view.js

```diff
@@ -64,31 +60,19 @@ AlbumsView.prototype.render = function() {
 };
 
 AlbumsView.prototype.getAlbums = function() {
-  return document.l10n.formatValues('unknownAlbum', 'unknown')
-    .then(([unknownAlbum, unknown]) => {
+  return document.l10n.formatValues('unknownAlbum', 'unknownArtist')
```

Comment:
I'm not sure I understand why unknownArtist was added in this patch. Is that just another bug that you're fixing at the same time here?